plugins/SitesManager/Controller.php

```diff
@@ -33,8 +35,29 @@ public function index()
         return $this->renderTemplate('index');
     }
 
-    public function getGlobalSettings() {
+    public function getCustomTypeSettings()
+    {
+        $idSite = Common::getRequestVar('idSite', 0, 'int');
```

Comment:
We render this here server side with a little delay for simplicity. Otherwise we'll have to create an API that returns information for all settings and we need to render pluginSettings and propertySettings client side etc. We will change this soon and do everything client side